### DISCUSSION TITLE
How do i start

### DISCUSSION THREAD

**Discussion by heloonihao** [free-audio/clap#417] [Q&A] [answered]
shoud i start on vst3?
i am new on audio development，
but cleveraudio.org tell me go to https://nakst.gitlab.io/tutorial/clap-part-1.html,
but there say "This tutorial is geared towards people who already have experience writing audio plugins"
i am search on web all is point to cleveraudio.org

**Comment by baconpaul** [comment]
H
This GitHub is for discussions about the evolution of the clap standard and is not a good resource for questions when just starting. 

If you have successfully written a clap the clap-wrapper project here allows you to make a vst3

but if your question is just more “how do I get started with audio dev” then I strongly recommend “the audio programmer” community which has loads of resources and a great discord to help people on their journey 

https://www.theaudioprogrammer.com/

**Reply by heloonihao** [reply]
thanks you take time off work,but in my country cant access google service,still thanks

**Reply by baconpaul** [reply, marked as answer]
Ahhh understood 

well if you want to write audio plugins and can access GitHub there are some good examples you can find here. If your goal is plugins you can download the juce framework which is on GitHub and has most of what you need for a basic starting point and comes with examples
